### PR TITLE
New Keyboard(s): Red Scarf II+ RS68 and RS78

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Leveraging off of @Maartenwut code that he sent me. 

I put in support for the RS68 (VerC) which uses the same matrix as the RS78. 

Filing the PR now just to separate more of the changes. Will still need to put in VerA and VerD along with lighting support. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

This fixes PR #1025 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/redscarf_iiplus/verb/config.h [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+// /*== customize breathing effect ==*/
+//   /*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//   #define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+//   /*==== use exp() and sin() ====*/
+//   #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//   #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (fauxpark) [inline keyboards/redscarf_iiplus/verb/config.h]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/redscarf_iiplus/verc/config.h [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+// /*== customize breathing effect ==*/
+//   /*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//   #define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+//   /*==== use exp() and sin() ====*/
+//   #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//   #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (fauxpark) [inline keyboards/redscarf_iiplus/verc/config.h]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/redscarf_iiplus/verc/config.h [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    RedScarf
+#define PRODUCT         RedScarfII+
+#define DESCRIPTION     QMK Firmware for the RedScarf II+
+
+/* key matrix size */
+#define MATRIX_ROWS 6
```
Comment (noroadsleft) [inline keyboards/redscarf_iiplus/verc/config.h]:
```suggestion
#define MATRIX_ROWS 5
```

File: keyboards/redscarf_iiplus/verc/verc.h [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	K30,      K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D,      K3E, \
+	K40, K41, K42,                K49,           K4A, K4B, K47, K4C, K4D,      K4E  \
+) \
+{ \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D, K2E }, \
+    { K30, XXX, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D, K3E }, \
+    { K40, K41, K42, XXX, XXX, XXX, XXX, K47, XXX, K49, K4A, K4B, K4C, K4D, K4E }, \
+    { XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX }  \
```
Comment (noroadsleft) [inline keyboards/redscarf_iiplus/verc/verc.h]:
```suggestion
```

File: keyboards/redscarf_iiplus/verc/verc.h [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D,      K3E, \
+	K40, K41, K42,                K49,           K4A, K4B, K47, K4C, K4D,      K4E  \
+) \
+{ \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D, K2E }, \
+    { K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D, K3E }, \
+    { K40, K41, K42, XXX, XXX, XXX, XXX, K47, XXX, K49, K4A, K4B, K4C, K4D, K4E }, \
+    { XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX, XXX }  \
```
Comment (noroadsleft) [inline keyboards/redscarf_iiplus/verc/verc.h]:
```suggestion
```

**Comment by mechmerlin** [comment]
> Ver.C doesn't use row 5 at all; seems to me you should be able to shrink its matrix. (Might require editing its matrix.c to quicken the loop, but that's above my skill set.)

I'm planning on decoupling the matrix.c and matrix definitions in a future PR to be more generic across all these boards. Id rather not make this a "custom electrical matrix".

**Comment by mechmerlin** [comment]
This is actually not a LAYOUT_65_ansi layout, I'm going to have to poke at the PCB a bit and change the names here.

**Comment by noroadsleft** [comment]
> This is actually not a LAYOUT_65_ansi layout, I'm going to have to poke at the PCB a bit and change the names here.

Split Backspace seems to be the only difference, so the layout is supported in hardware, just not in the code. I'm fine with this for your purposes; I can tackle it later if desired.

**Comment by mechmerlin** [comment]
All should be good now. =)

**Pull request merged**